source/features/open-all-notifications.tsx

```diff
@@ -128,3 +128,13 @@ void features.add(import.meta.url, {
 	],
 	init,
 });
+
+/*
+
+Test URLs:
+	
+https://github.com/notifications (Grouped by date)
+https://github.com/notifications (Grouped by repo)
+https://github.com/notifications?query=reason%3Acomment (which is an unsaved filter)
+
+*/
```

Comment:
Without this:

<img width="534" alt="Screenshot" src="https://github.com/refined-github/refined-github/assets/1402241/8cd3f661-6ec0-4474-8bb5-3c43417015aa">


Thanks to:

- https://github.com/refined-github/refined-github/pull/6653